rmw_fastrtps_cpp/src/functions.cpp

```diff
@@ -507,7 +507,7 @@ class WriterInfo : public ReaderListener
   {
     WriterProxyData proxyData;
     if (change->kind == ALIVE) {
-      CDRMessage_t tempMsg;
+      CDRMessage_t tempMsg(0);
```

Comment:
I ended up opening https://github.com/eProsima/Fast-RTPS/issues/115, since it seems this could be made a lot nicer to use.